src/components/dashboard/shared/VerticalTimeline.js

```diff
@@ -16,24 +16,42 @@ export default class VerticalTimeline extends Component {
   sortItems = memoize((items) => items.sort(((a, b) => moment(b.date) - moment(a.date))));
 
   handleViewMore = () => {
-    const { items, viewMoreCount } = this.props;
+    const { items, viewCount } = this.props;
     const { displayCount  } = this.state;
-    let newDisplayCount = displayCount + viewMoreCount;
+    let newDisplayCount = displayCount + viewCount;
     if (newDisplayCount > items.length) { newDisplayCount = items.length; }
 
     this.setState({ displayCount: newDisplayCount });
   }
 
+  handleViewLess = () => {
+    const { initialDisplayCount, viewCount } = this.props;
+    const { displayCount  } = this.state;
+    let newDisplayCount = displayCount - viewCount;
+    if (newDisplayCount < viewCount) { newDisplayCount = initialDisplayCount; }
```

Comment:
Almost: `if (newDisplayCount < initialDisplayCount) { newDisplayCount = initialDisplayCount; }`. Maybe copy and paste?